aiidalab_widgets_base/computational_resources.py

```diff
@@ -904,8 +954,13 @@ def _observe_computer_setup(self, _=None):
             return
         if "setup" in self.computer_setup:
             for key, value in self.computer_setup["setup"].items():
-                if hasattr(self, key):
+                if key == "default_memory_per_machine":
+                    self.default_memory_per_machine_widget.value = int(
+                        value / 1024**2
```

Comment:
I don't understand why the 1024 value is squared?

Reply:
The value in the database is in kB, but we agreed previously to keep GB for the widget. Just to answer why 1024 is squared.

I also noticed that this line is wrong, though. Thanks, @danielhollas, for pointing at it. The values shouldn't be an integer but a string. 

Possible solutions are:
```python
self.default_memory_per_machine_widget.value =
         str(decimal.Decimal(f"{value/1024**2:0.3f}").normalize()) + ' GB'
```
This requires importing `decimal`.

Another solution is simply to say that the imported value is in kB
```python
f"{value KB}
```

Any approach is good for me.

I am going to disappear, for some time, so I let @unkcpz and/or @danielhollas pick up an appropriate solution.

Reply:
I prefer the second one without the `decimal` dependency.